PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutRequest.java

```diff
@@ -1,7 +1,18 @@
 package com.braintreepayments.api;
 
-/**
- * Represents the parameters that are needed to start the PayPal Native Checkout flow
- */
-public class PayPalNativeCheckoutRequest {
+public class PayPalNativeCheckoutRequest extends PayPalCheckoutRequest {
+
+    /**
+     * @param amount The transaction amount in currency units (as * determined by setCurrencyCode).
+     *               For example, "1.20" corresponds to one dollar and twenty cents. Amount must be a non-negative
+     *               number, may optionally contain exactly 2 decimal places separated by '.', optional
+     *               thousands separator ',', limited to 7 digits before the decimal point.
```

Comment:
```suggestion
     *               number, may optionally contain exactly 2 decimal places separated by '.'
     *               and is limited to 7 digits before the decimal point.
```
This doc string has been updated slightly on `PayPalCheckoutRequest` and the change should apply here also